teuthology/task/install/rpm.py

```diff
@@ -149,7 +150,10 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
         # install it separately because, otherwise, ceph-test cannot be
         # installed (even with --force) when there are several conflicting
         # repos from different vendors.
-        rpm = ["librados2", "ceph-test"]
+        if dist_release in ['opensuse', 'sle'] and dist_version >= 15 and dist_version < 42:
```

Comment:
Do you mean for 15 versions it should take _librados3_ and for 42.x use _librados2_? If so, why not to check `dist_version[0] == 15` then?

Reply:
because librados3 will be used from 15 on up, but not for 42 :-)